### PR TITLE
Release/4.0.0-beta.6

### PR DESCRIPTION
**Pull request by taquitos**
## 4.0.0-beta.6
- Sixth beta for RevenueCat framework 🎉
    100% Swift framework + ObjC support.
- See our [RevenueCat V4 API update doc](docs/V4_API_Updates.md) for API updates.
- Add more specific backend error subcodes
    https://github.com/RevenueCat/purchases-ios/pull/927

### CODE REVIEW

**Review by beylmk — APPROVED** [review]
6️⃣

File: CHANGELOG.latest.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
...
 - See our [RevenueCat V4 API update doc](docs/V4_API_Updates.md) for API updates.
-- Improve error handling for backend errors
-    https://github.com/RevenueCat/purchases-ios/pull/922
-- Replaced #file with #fileID
-    https://github.com/RevenueCat/purchases-ios/pull/921
-- Rename a few public APIs to reduce confusion
-    https://github.com/RevenueCat/purchases-ios/pull/917
\ No newline at end of file
+- Add more specific backend error subcodes
+    https://github.com/RevenueCat/purchases-ios/pull/927
```
Comment (beylmk) [inline CHANGELOG.latest.md:5]: we probably want to make this change in android too, will add to the rcv5 list...

and do we want to update the error code docs with some info on these subcodes?

File: IntegrationTests/CarthageIntegration/CarthageIntegration.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -476,7 +476,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```
Comment (beylmk) [inline IntegrationTests/CarthageIntegration/CarthageIntegration.xcodeproj/project.pbxproj:479]: 🙏  thank you!

File: CHANGELOG.latest.md (hunk 1 above)
Comment (taquitos) [reply CHANGELOG.latest.md:5]: Awesome, thank you, and yeah, I think adding a short blurb that these errors are mostly for us to help debug any developer issues. I can add a task for that, too

File: IntegrationTests/CarthageIntegration/CarthageIntegration.xcodeproj/project.pbxproj (hunk 2 above)
Comment (taquitos) [reply IntegrationTests/CarthageIntegration/CarthageIntegration.xcodeproj/project.pbxproj]: nodding

**Pull request merged**